src/vendor.ts

```diff
@@ -1 +1,2 @@
+import '../node_modules/hammerjs/hammer.min.js';
```

Comment:
`import 'hammerjs';` should suffice. Cannot be `Muuri` imported the same way?

Reply:
Tried. Typescript didn't like it.